### PR TITLE
Onvember/doc build

### PR DESCRIPTION
**Pull request by Onvember**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+    luatest test/feature_test.lua         # by file
+    luatest test/integration              # all within directory
+    luatest feature other.test_example_2  # by group or test name
+    luatest --help # list available options
+
+Luatest automatically requires ``test/helper.lua`` file if it's present.
+You can configure luatest or run any bootstrap code there.
+
+See test directory template in
+`cartridge-cli repo <https://github.com/tarantool/cartridge-cli/tree/master/templates/cartridge/test>`_
```
Comment (rosik) [inline README.rst:85]: 404 page not found

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+    local t = require('luatest')
+    local g = t.group('other')
+    -- ...
+    g.test_example_2 = function() ... end
+    g.test_example_m = function() ... end
+
+
+Run them.
+
+.. code-block:: none
```
Comment (rosik) [inline README.rst:73]:
```suggestion
.. code-block:: bash
```

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+
+    luatest                               # all in ./test directory
+    luatest test/feature_test.lua         # by file
+    luatest test/integration              # all within directory
+    luatest feature other.test_example_2  # by group or test name
+    luatest --help # list available options
```
Comment (rosik) [inline README.rst:79]:
```suggestion
    luatest --help                        # list available options
    luatest                               # run all in ./test directory
    luatest test/feature_test.lua         # run test by file
    luatest test/integration              # run all within directory
    luatest feature other.test_example_2  # run by group or test name
```

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+Tool for testing tarantool applications. (`Build Status <https://travis-ci.com/tarantool/luatest.svg?branch=master)](https://travis-ci.com/tarantool/luatest>`_).
+
+Highlights:
+
+- executable to run tests in directory or specific files,
+- before/after suite hooks,
+- before/after test group hooks,
+- :ref:`output capturing <capturing-output>`,
+- :ref:`helpers <test-helpers>` for testing tarantool applications,
+- :ref:`luacov integration <luacov-integration>`.
```
Comment (rosik) [inline README.rst:14]: Github doesn't render it properly.

File: README.rst (hunk 1 above)
Comment (Onvember) [reply README.rst]: fixed in 07dfb7e

File: README.rst (hunk 2 above)
Comment (Onvember) [reply README.rst]: fixed in 07dfb7e

File: README.rst (hunk 3 above)
Comment (Onvember) [reply README.rst]: fixed in 07dfb7e

File: README.rst (hunk 4 above)
Comment (Onvember) [reply README.rst]: fixed in 07dfb7e

**Review by rosik — APPROVED** [review]

**Pull request merged**